### ISSUE TITLE
Can't use GtkSharp on macOS

### ISSUE THREAD

**Issue by monoluser** [GtkSharp/GtkSharp#78]
Good day!

I have installed the gtk3 package via MacPorts and run the following commands:
```
$ dotnet new sln -n HelloGTK
$ dotnet new gtkapp -o HelloGTK
$ dotnet sln HelloGTK.sln add HelloGTK
$ cd HelloGTK
$ dotnet run
```
On Linux as well as Windows, the sample Window shows up but on macOS, the following exception is thrown:
```
Unhandled Exception: System.TypeInitializationException: The type initializer for 'Gtk.Application' threw an exception. ---> System.DllNotFoundException: Gtk
   at GLibrary.Load(Library library)
   at Gtk.Application..cctor()
   --- End of inner exception stack trace ---
   at Gtk.Application.Init()
   at HelloGTK.Program.Main(String[] args) in /Users/luser/dotnets/HelloGTK/Program.cs:line 11
```
Any hints for making Gtk being found?

**Comment by pedrohex** [comment]
Hi there,

I solve this issue by installing gtk3 on mac.

I used brew:

brew install gtk+3

Cheers

On 2 Mar 2019, at 00:13, monoluser <notifications@github.com<mailto:notifications@github.com>> wrote:


Good day!

I have installed the gtk3 package via MacPorts and run the following commands:

$ dotnet new sln -n HelloGTK
$ dotnet new gtkapp -o HelloGTK
$ dotnet sln HelloGTK.sln add HelloGTK
$ cd HelloGTK
$ dotnet run


On Linux as well as Windows, the sample Window shows up but on macOS, the following exception is thrown:

Unhandled Exception: System.TypeInitializationException: The type initializer for 'Gtk.Application' threw an exception. ---> System.DllNotFoundException: Gtk
   at GLibrary.Load(Library library)
   at Gtk.Application..cctor()
   --- End of inner exception stack trace ---
   at Gtk.Application.Init()
   at HelloGTK.Program.Main(String[] args) in /Users/luser/dotnets/HelloGTK/Program.cs:line 11


Any hints for making Gtk being found?
